core/src/avm2/activation.rs

```diff
@@ -533,6 +534,7 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
             );
         }
 
+        let current_position = reader.pos(full_data);
```

Comment:
I feel like this should be called `instruction_start` instead; because by the time `op_lookup_switch` runs, it's no longer the current position (and we grab the position again so we can subtract out the length of the instruction from the offsets it provides).

Reply:
(This should, obviously, also apply to `current_pos` in the arguments to `op_lookup_switch`).

Reply:
Done